PHP/CalculateMerchantApiHash.php

```diff
@@ -49,15 +49,15 @@
 ]);
 /* This will output following string with line break character \n
 POST
-https://api.paytrail.com/merchant/v1/payments/102402728626/refunds
+/payments/102402728626/refunds

```

Comment:
Should this be in the format `/merchant/v1/payments/:id/refunds` like described in the [documentation](https://docs.paytrail.com/merchant-api/v1/accessing/)?